src-built-in/components/appLauncher/src/stores/appLauncherStore.js

```diff
@@ -260,6 +259,8 @@ var Actions = {
 			pins.push(thePin);
 		}
 
+		var componentTypeDotRemove = displayName.replace(/[.]/g, "^DOT^");
```

Comment:
Changed from componentType to displayName. This allows pinning to work with the display name, not the component type.